engineio/Cargo.toml

```diff
@@ -14,7 +14,7 @@ license = "MIT"
 all-features = true
 
 [dependencies]
-base64 = "0.21.5"
```

Comment:
Please **don't** do unrelated changes as part of your PRs! Why does this needs to be bumped (also to the next major version, which implies a breaking change).